packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.html

```diff
@@ -25,3 +14,14 @@
     <div class="nui-step-optional" *ngIf="optional && state != 'error'">{{optionalLabel}}</div>
     <div class="nui-step-sub-label-error" *ngIf="state == 'error'">{{errorMessage}}</div>
 </div>
+
+<div class="nui-step-icon-state-{{state}} nui-wizard-step-icon"
+     [class.nui-wizard-step-header__icon--selected]="selected">
```

Comment:
This looks like it could be duplicate functionality with what's happening on line 18 with the `state` interpolation. Or, maybe the `selected` state styling logic could be merged with the styling logic for the other states somehow?